omero/sysadmins/cli/usergroup.rst

```diff
@@ -57,6 +57,33 @@ below:
 
 .. literalinclude:: /downloads/ldap/setdn.out
 
+
+User deactivation
+^^^^^^^^^^^^^^^^^
+
+To deactivate a user, remove him/her from the system group ``user``.
+Use the command :program:`omero user leavegroup` and specify the ``user`` group as the target::
+
+    # Remove jsmith from group user
+    $ bin/omero user leavegroup user --name=jsmith
+
+To reactivate the user, add him/her back to the system group ``user`` i.e.::
+
+    $ bin/omero group joingroup jsmith --name=user
```

Comment:
`bin/omero user joingroup user --name=jsmith`

Reply:
Both work but I see your point

Reply:
```
$ /opt/ome1/dist/bin/omero group joingroup
...
/opt/ome1/dist/bin/omero group: error: argument <subcommand>: invalid choice: 'joingroup'
```

Reply:
I see what I did. Looking back at my history
I ran the correct command and only partially update the doc
``group joingroup`` does not work